### PR TITLE
added support for institutions

### PR DESCRIPTION
**Pull request by pazwicker**
Adds support for pulling institution data from Monarch.

Institution data can be joined back to account data to add more information to institution sync status, connection details, and more.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -21,6 +21,11 @@ def main() -> None:
     with open("data.json", "w") as outfile:
         json.dump(accounts, outfile)
 
+    # Institutions
+    institutions = asyncio.run(mm.get_institutions())
+    with open("insitutions.json", "w") as outfile:
```
Comment (hammem) [inline main.py:26]: Nit: spelling on 'institution'

File: monarchmoney/monarchmoney.py [hunk 2]
```diff
@@ -489,6 +489,93 @@ async def get_account_holdings(self, account_id: int) -> Dict[str, Any]:
             variables=variables,
         )
 
+    async def get_institutions(self) -> Dict[str, Any]:
+        """
+        Gets insitution data from the account.
```
Comment (hammem) [inline monarchmoney/monarchmoney.py:494]: Nit: spelling

**Review by hammem — CHANGES_REQUESTED** [review]
Thanks, @pazwicker !  Just a small thing for spelling on 'institution' in a couple of places.

File: main.py (hunk 1 above)
Comment (pazwicker) [reply main.py:26]: Thanks for the catch!

File: monarchmoney/monarchmoney.py (hunk 2 above)
Comment (pazwicker) [reply monarchmoney/monarchmoney.py:494]: And again!

**Review by hammem — APPROVED** [review]
Awesome!  Thanks again for the contributions. The library is growing in functionality so much faster with all the help!

**Pull request merged**